docs/release-notes/2.41.2.md

```diff
@@ -0,0 +1,117 @@
+
+
+
+#### PostgreSQL Checkpoints, Buffers and WAL Usage dashboard
+
+In our continuous effort to enhance our dashboards, we have introduced this new experimental dashboard that provides more data about your PostgreSQL instances.
+
+By leveraging this data, administrators can gain insights into their PostgreSQL servers and fine-tune their performance. Understanding and managing checkpoints, buffers, and WAL usage contribute to a well-performing and reliable PostgreSQL environment.
+
+If you’re looking for in-depth insights into this dashboard, refer to our [blog post](https://www.percona.com/blog/postgresql-checkpoints-buffers-and-wal-usage-with-percona-monitoring-and-management/).
+
+![!image](../_images/pg_checkpoints_buffers_wal_dashboard.png)
+
+#### PostgreSQL Patroni dashboard
+
+Starting with PMM 2.41.2, we have included a new dashboard designed to monitor [Patroni](https://patroni.readthedocs.io/en/latest/) as an external service. For more information, see our documentation on [external services](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/external.html).
+
+This dashboard facilitates gathering more data for your PostgreSQL cluster inside PMM. 
+
+Read our [blog post](https://www.percona.com/blog/monitoring-a-postgresql-patroni-cluster/) to learn more about this dashboard and how to add Patroni monitoring.
+
+![!image](../_images/patroni_dashboard.png)
+
+!!! caution alert alert-warning "Important"
+    We would appreciate your feedback on these experimental dashboards to help us enhance it further.
+
+
+## Improvements
+
+
+[PMM-10974](https://perconadev.atlassian.net/browse/PMM-10974) - By default, PMM used `pg_stat_statements` for **stat tracking**, but now it uses `pg_stat_monitor`, which provides additional metrics and supports **explains/examples** functionality that `pg_stat_statements` does not.
+
+[PMM-12884](https://perconadev.atlassian.net/browse/PMM-12884) - In PMM dump, the **Address** field on the **Send to Support** screen now comes with the address and port pre-filled, making it easier to send logs to support.
+
+[PMM-12887](https://perconadev.atlassian.net/browse/PMM-12887) - [Tech Preview] This new experimental dashboard has been added in PMM 2.41.2 for monitoring PostgreSQL checkpoints, buffers and WAL usage.
+
+[PMM-12888](https://perconadev.atlassian.net/browse/PMM-12888) - [Tech Preview] This new experimental dashboard has been added in PMM 2.41.2 to monitor [Patroni](https://patroni.readthedocs.io/en/latest/) as an external service.
+
+[PMM-12960](https://perconadev.atlassian.net/browse/PMM-12960) - [Tech Preview] The latest PMM release includes redesigned Kubernetes dashboards with improved clarity and usability.
+
+[PMM-12806](https://perconadev.atlassian.net/browse/PMM-12806)- In prior versions, PMM Server would prioritize certain VictoriaMetrics command line flags over environment variables. This behavior has now been addressed.
```

Comment:
```suggestion
[PMM-12806](https://perconadev.atlassian.net/browse/PMM-12806)- In prior versions, PMM Server had certain non-configurable Victoria Metrics settings with default values. This behavior has now been addressed.
```

Reply:
and it's in both lists: improvements and bugs

Reply:
Fixed. Added it in improvements.